include/tudat/simulation/estimation_setup/processTrackingTxtFile.h

```diff
@@ -0,0 +1,319 @@
+/*    Copyright (c) 2010-2023, Delft University of Technology
+ *    All rights reserved
+ *
+ *    This file is part of the Tudat. Redistribution and use in source and
+ *    binary forms, with or without modification, are permitted exclusively
+ *    under the terms of the Modified BSD license. You should have received
+ *    a copy of the license with this file. If not, please or visit:
+ *    http://tudat.tudelft.nl/LICENSE.
+ *
+ *    References: 820-013, TRK-2-18 Tracking System Interfaces Orbit Data File Interface, Revision E, 2008, JPL/DSN
+ */
+
+
+#ifndef TUDAT_PROCESSTRACKINGTXTFILE_H
+#define TUDAT_PROCESSTRACKINGTXTFILE_H
+
+#include "tudat/basics/utilities.h"
+#include "tudat/io/readTrackingTxtFile.h"
+#include "tudat/astro/observation_models/observableTypes.h"
+#include "tudat/io/basicInputOutput.h"
+#include "tudat/astro/basic_astro/timeConversions.h"
+#include "tudat/astro/ground_stations/transmittingFrequencies.h"
+#include "tudat/simulation/estimation_setup/observations.h"
+#include "tudat/simulation/estimation_setup/observationSimulationSettings.h"
+#include "tudat/simulation/environment_setup/body.h"
+#include "tudat/math/interpolators/lookupScheme.h"
+
+// TODO:
+//  Ancillary settings (frequency, time scale, ...)
+//  Format checks
+//  Unit Checks
+//  Add time conversions
+
+
+
+namespace tudat
+{
+namespace observation_models
+{
+
+//! Utility function that might belong elsewhere
+template< typename TimeScalarType = double >
+TimeScalarType convertCalendarDateToJulianDaySinceJ2000(const int calendarYear,
```

Comment:
Why does this function needs to be added? If the reason is to have a specific function that "specializes" convertCalendarDateToJulianDaysSinceEpoch for a J2000 reference epoch, that's a useful thing to have. But, then please move it into the timeConversions.h file

Reply:
Indeed. I will move it.